roles/dalite-server/templates/default.site

```diff
@@ -0,0 +1,37 @@
+server {
+       listen         80;
+       server_name    {{ DALITE_SERVER_DOMAIN }};
+       return         301 https://$server_name$request_uri;
+}
+
+server {
+   listen         443 ssl;
+   server_name    {{ DALITE_SERVER_DOMAIN }};
+
+   add_header Strict-Transport-Security "max-age=31536000";
```

Comment:
I added this kind of automatically, but it would be good to have confirmation it makes sense: https://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security .